src/components/calculator.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculator from './calculator';
+
+it('Calculator component is working', () => {
+  const tree = renderer.create(<Calculator />).toJSON();
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] Kindly stimulate user interaction by using the react testing library to fire events and test the values returned from calculations.

Reply:
Done it.